x/ccv/provider/keeper/grpc_query.go

```diff
@@ -93,6 +125,19 @@ func (k Keeper) GetConsumerChain(ctx sdk.Context, consumerId string) (types.Chai
 		strDenylist[i] = addr.String()
 	}
 
+	phase, ok := k.GetConsumerPhase(ctx, consumerId)
+	if !ok {
+		return types.Chain{}, fmt.Errorf("cannot find phase for consumer (%s)", consumerId)
+	}
+
+	metadata, err := k.GetConsumerMetadata(ctx, consumerId)
+	if err != nil {
+		return types.Chain{}, fmt.Errorf("cannot get metadata for consumer (%s): %w", consumerId, err)
+	}
+
+	clientID, _ := k.GetConsumerClientId(ctx, consumerId)
```

Comment:
Since those appear first in the `types.Chain`, let's retrieve them in the same order (?)